### PR TITLE
[RHPAM-4111]-Certify Spring Boot 2.4.x

### PR DESCRIPTION
**Pull request by VaniHaripriya**
JIRA: https://issues.redhat.com/browse/RHPAM-4111
Upgraded version of narayana spring boot starter to align with Red Hat supported Spring Boot 2.4.9 version.
This PR should be merged along with - https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1860

### CODE REVIEW

**Comment by gmunozfe** [comment]
jenkins run fdb

**Review by gmunozfe — APPROVED** [review]
Looks good to me, just waiting for the CI to be green.
@VaniHaripriya great work!

File: kie-spring-boot/pom.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
   </modules>
 
   <properties>
-    <version.me.snowdrop.narayana>2.4.1</version.me.snowdrop.narayana>
+    <version.me.snowdrop.narayana>2.6.1.redhat-00001</version.me.snowdrop.narayana>
```
Comment (mareknovotny) [inline kie-spring-boot/pom.xml:36]: don't use redhat specific versions here please. just community ones without redhat-#### suffix

**Review by mareknovotny — CHANGES_REQUESTED** [review]
We need to use only community deps here

**Comment by mareknovotny** [comment]
there is no need to run FDB here, it requires to be run in top PR so in  https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1860

File: kie-spring-boot/pom.xml (hunk 1 above)
Comment (gmunozfe) [reply kie-spring-boot/pom.xml:36]: @mareknovotny this comes from the fact that dependencies should be aligned with ROAR, which declares that one:
https://download.eng.brq.redhat.com/released/JBoss-middleware/rhoar/spring-boot/2.4.9/extras/repository-artifact-list.txt

Is it the same without redhat-#### suffix?

**Comment by gmunozfe** [comment]
An enforcer rule is failing:

```
[2022-01-11T15:00:57.916Z] Duplicate classes found:
[2022-01-11T15:00:57.916Z] 
[2022-01-11T15:00:57.916Z]   Found in:
[2022-01-11T15:00:57.916Z]     org.apache.geronimo.specs:geronimo-jms_2.0_spec:jar:1.0-alpha-2:compile
[2022-01-11T15:00:57.916Z]     jakarta.jms:jakarta.jms-api:jar:2.0.3:compile
[2022-01-11T15:00:57.916Z]   Duplicate classes:
[2022-01-11T15:00:57.916Z]     javax/jms/QueueConnection.class
[2022-01-11T15:00:57.916Z]     javax/jms/MessageNotReadableException.class
[2022-01-11T15:00:57.916Z]     javax/jms/XAJMSContext.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSDestinationDefinitions.class
[2022-01-11T15:00:57.916Z]     javax/jms/InvalidDestinationRuntimeException.class
[2022-01-11T15:00:57.916Z]     javax/jms/ObjectMessage.class
[2022-01-11T15:00:57.916Z]     javax/jms/TopicRequestor.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSException.class
[2022-01-11T15:00:57.916Z]     javax/jms/InvalidSelectorException.class
[2022-01-11T15:00:57.916Z]     javax/jms/Message.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSConnectionFactoryDefinition.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSSecurityException.class
[2022-01-11T15:00:57.916Z]     javax/jms/IllegalStateRuntimeException.class
[2022-01-11T15:00:57.916Z]     javax/jms/ServerSessionPool.class
[2022-01-11T15:00:57.916Z]     javax/jms/Topic.class
[2022-01-11T15:00:57.916Z]     javax/jms/ConnectionFactory.class
[2022-01-11T15:00:57.916Z]     javax/jms/ConnectionMetaData.class
[2022-01-11T15:00:57.916Z]     javax/jms/MessageListener.class
[2022-01-11T15:00:57.916Z]     javax/jms/XAConnectionFactory.class
[2022-01-11T15:00:57.916Z]     javax/jms/Destination.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSConnectionFactoryDefinitions.class
[2022-01-11T15:00:57.916Z]     javax/jms/MessageConsumer.class
[2022-01-11T15:00:57.916Z]     javax/jms/QueueRequestor.class
[2022-01-11T15:00:57.916Z]     javax/jms/QueueBrowser.class
[2022-01-11T15:00:57.916Z]     javax/jms/CompletionListener.class
[2022-01-11T15:00:57.916Z]     javax/jms/XASession.class
[2022-01-11T15:00:57.916Z]     javax/jms/IllegalStateException.class
[2022-01-11T15:00:57.916Z]     javax/jms/TransactionInProgressException.class
[2022-01-11T15:00:57.916Z]     javax/jms/MessageFormatRuntimeException.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSContext.class
[2022-01-11T15:00:57.916Z]     javax/jms/InvalidSelectorRuntimeException.class
[2022-01-11T15:00:57.916Z]     javax/jms/ResourceAllocationException.class
[2022-01-11T15:00:57.916Z]     javax/jms/XATopicConnectionFactory.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSConnectionFactory.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSPasswordCredential.class
[2022-01-11T15:00:57.916Z]     javax/jms/StreamMessage.class
[2022-01-11T15:00:57.916Z]     javax/jms/Queue.class
[2022-01-11T15:00:57.916Z]     javax/jms/ResourceAllocationRuntimeException.class
[2022-01-11T15:00:57.916Z]     javax/jms/JMSDestinationDefinition.class
[2022-01-11T15:00:57.917Z]     javax/jms/MessageNotWriteableException.class
[2022-01-11T15:00:57.917Z]     javax/jms/DeliveryMode.class
[2022-01-11T15:00:57.917Z]     javax/jms/TopicConnection.class
[2022-01-11T15:00:57.917Z]     javax/jms/MessageFormatException.class
[2022-01-11T15:00:57.917Z]     javax/jms/XAQueueConnection.class
[2022-01-11T15:00:57.917Z]     javax/jms/XAConnection.class
[2022-01-11T15:00:57.917Z]     javax/jms/MessageProducer.class
[2022-01-11T15:00:57.917Z]     javax/jms/TemporaryQueue.class
[2022-01-11T15:00:57.917Z]     javax/jms/TopicConnectionFactory.class
[2022-01-11T15:00:57.917Z]     javax/jms/XAQueueSession.class
[2022-01-11T15:00:57.917Z]     javax/jms/InvalidClientIDException.class
[2022-01-11T15:00:57.917Z]     javax/jms/BytesMessage.class
[2022-01-11T15:00:57.917Z]     javax/jms/QueueSender.class
[2022-01-11T15:00:57.917Z]     javax/jms/TransactionRolledBackException.class
[2022-01-11T15:00:57.917Z]     javax/jms/JMSSessionMode.class
[2022-01-11T15:00:57.917Z]     javax/jms/Connection.class
[2022-01-11T15:00:57.917Z]     javax/jms/MessageEOFException.class
[2022-01-11T15:00:57.917Z]     javax/jms/JMSRuntimeException.class
[2022-01-11T15:00:57.917Z]     javax/jms/ExceptionListener.class
[2022-01-11T15:00:57.917Z]     javax/jms/ConnectionConsumer.class
[2022-01-11T15:00:57.917Z]     javax/jms/ServerSession.class
[2022-01-11T15:00:57.917Z]     javax/jms/JMSSecurityRuntimeException.class
[2022-01-11T15:00:57.917Z]     javax/jms/MessageNotWriteableRuntimeException.class
[2022-01-11T15:00:57.917Z]     javax/jms/QueueSession.class
[2022-01-11T15:00:57.917Z]     javax/jms/TemporaryTopic.class
[2022-01-11T15:00:57.917Z]     javax/jms/TopicPublisher.class
[2022-01-11T15:00:57.917Z]     javax/jms/XAQueueConnectionFactory.class
[2022-01-11T15:00:57.917Z]     javax/jms/QueueReceiver.class
[2022-01-11T15:00:57.917Z]     javax/jms/MapMessage.class
[2022-01-11T15:00:57.917Z]     javax/jms/InvalidClientIDRuntimeException.class
[2022-01-11T15:00:57.917Z]     javax/jms/TransactionRolledBackRuntimeException.class
[2022-01-11T15:00:57.917Z]     javax/jms/Session.class
[2022-01-11T15:00:57.917Z]     javax/jms/TextMessage.class
[2022-01-11T15:00:57.917Z]     javax/jms/TopicSession.class
[2022-01-11T15:00:57.917Z]     javax/jms/TopicSubscriber.class
[2022-01-11T15:00:57.917Z]     javax/jms/InvalidDestinationException.class
[2022-01-11T15:00:57.917Z]     javax/jms/TransactionInProgressRuntimeException.class
[2022-01-11T15:00:57.917Z]     javax/jms/JMSProducer.class
[2022-01-11T15:00:57.917Z]     javax/jms/JMSConsumer.class
[2022-01-11T15:00:57.917Z]     javax/jms/QueueConnectionFactory.class
[2022-01-11T15:00:57.917Z]     javax/jms/XATopicConnection.class
[2022-01-11T15:00:57.917Z]     javax/jms/XATopicSession.class
[2022-01-11T15:00:57.917Z] 
```

File: kie-spring-boot/pom.xml (hunk 1 above)
Comment (mareknovotny) [reply kie-spring-boot/pom.xml:36]: yes, that productized version is orchestrated automatically in build if it exists

Comment (VaniHaripriya) [reply kie-spring-boot/pom.xml:36]: Sure, will update accordingly.

**Comment by mareknovotny** [comment]
jenkins retest this please

**Comment by VaniHaripriya** [comment]
jenkins retest this please

**Comment by VaniHaripriya** [comment]
jenkins retest this

**Comment by VaniHaripriya** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by VaniHaripriya** [comment]
jenkins retest this

**Comment by VaniHaripriya** [comment]
The failures are unrelated to this change.

**Comment by gmunozfe** [comment]
> The failures are unrelated to this change.

Launched a PR https://github.com/kiegroup/droolsjbpm-integration/pull/2691 to fix those failures

**Comment by mareknovotny** [comment]
jenkins retest this please

#2691 merged so testing again

**Pull request closed without merging**